### PR TITLE
chore(docs): improve Docker instructions

### PR DESCRIPTION
**Pull request by mcnaveen**
## Why this PR?
- This PR improves the instructions for running Documenso in docker

### CODE REVIEW

**Comment by Mythie** [comment]
Hey there, it looks like you haven't accepted our contributor license agreement yet. In order for us to accept your pull request we ask that you please fill out the CLA:

https://documen.so/cla

**Comment by mcnaveen** [comment]
Hi @Mythie I just did. Thanks :smile:

File: README.md [hunk 1]
```diff
@@ -217,8 +217,21 @@ For the digital signature of your documents you need a signing certificate in .p
 
 Want to create a production ready docker image? Follow these steps:
 
-- Run `./docker/build.sh` in the root directory.
-- Publish the image to your docker registry of choice.
+- cd into `docker` directory
+- Make `build.sh` executable by running `chmod +x build.sh`
```
Comment (Mythie) [inline README.md:221]: It should already be executable 🤔 Git preserves the permission bits afaik

Comment (mcnaveen) [reply README.md:221]: Not sure it fully does. Often faced similar issues on Linux.

[Ref](https://archive.kernel.org/oldwiki/git.wiki.kernel.org/index.php/ContentLimitations.html)

Comment (Mythie) [reply README.md:221]: > except for the executable bit

But if you're having issues it's probably good to include anyways!

**Review by Mythie — APPROVED** [review]

**Comment by Mythie** [comment]
Thanks for your contribution! 😇 

lgtm-b99

File: README.md (hunk 1 above)
Comment (mcnaveen) [reply README.md:221]: Great, Thank you 😄

**Pull request merged**